src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -123,14 +219,87 @@ const StreamGroup = createReactClass({
         {hasGuideAnchor && <GuideAnchor target="issue_stream" />}
         {withChart && (
           <Box width={160} mx={2} className="hidden-xs hidden-sm">
-            <GroupChart id={data.id} statsPeriod={statsPeriod} data={data} />
+            <GroupChart
+              id={data.id}
+              statsPeriod={statsPeriod}
+              data={data}
+              hasDynamicIssueCounts={hasDynamicIssueCounts}
+            />
           </Box>
         )}
         <Flex width={[40, 60, 80, 80]} mx={2} justifyContent="flex-end">
-          <StyledCount value={data.count} />
+          <Tooltip
+            disabled={!hasDynamicIssueCounts}
+            popperStyle={popperStyle}
+            isHoverable
+            title={
+              <TooltipContent>
+                {data.lifetime && (
+                  <TooltipRow>
+                    <TooltipCount>{data.lifetime.count}</TooltipCount>
+                    <TooltipText>{t('Since issue began')}</TooltipText>
+                  </TooltipRow>
+                )}
+                <TooltipRow>
+                  <TooltipCount>{data.count}</TooltipCount>
+                  <TooltipText>{t(`Within ${summary}`)}</TooltipText>
+                  <StyledIconTelescope to={this.getDiscoverUrl()} color={theme.blue300} />
```

Comment:
If you don't want to send users to a upsell state you'll need to feature gate these discover link as not all plans include discover.

Reply:
I will add that check here but if an org has dynamic issue counts feature they will likely have discovery as well I assume.